libcxx/src/memory.cpp

```diff
@@ -132,19 +133,11 @@ __sp_mut& __get_sp_mut(const void* p) {
 
 #endif // _LIBCPP_HAS_THREADS
 
+#if defined(_LIBCPP_ALIGN_DEFINE_LEGACY_INLINE_FUNCTIONS)
+
 void* align(size_t alignment, size_t size, void*& ptr, size_t& space) {
```

Comment:
Can we just add `_LIBCPP_EXPORTED_FROM_ABI` here and then avoid declaring this version in `<__memory/align.h>`?